xml_inscriptions/transcribed/MI.AA.UM.KM.L.1585.xml

```diff
@@ -150,7 +151,7 @@
                   </name> 
                   <name type="cognomen">
                      <expan>
-                        <abbr>Q</abbr></expan>
+                        <abbr>Q</abbr><ex/></expan>
```

Comment:
If you can't expand the abbreviation because you don't know what the expansion is, you don't put an empty `<ex>`; just leave it as it was with just `<abbr>Q</abbr>`, otherwise the markup is wrong (https://epidoc.stoa.org/gl/latest/trans-abbrevnotex.html)

Reply:
Ok got it. I think I added the empty element because <abbr>...</abbr> isn't showing up as an unexpanded abbreviation on the website for me. [Here](https://usepigraphy.brown.edu/projects/usep/inscription/MI.AA.UM.KM.L.1575/) the "i" should be showing up as i(---) or something correct? I think the unexpanded abbreviations were showing up correctly on the website at some point so I'm not sure what changed.